app/models/journal.rb

```diff
@@ -110,7 +110,7 @@ def journal_row_attributes_from_product_and_total(product, total)
   validate :must_have_order_details, :on => :create, :if => :order_details_for_creation
   validate :must_not_span_fiscal_years, :on => :create, :if => :has_order_details_for_creation?
   validate :journal_date_cannot_be_before_last_fulfillment, :on => :create, :if => :has_order_details_for_creation?
-
+  validates_with JournalDateMustBeAfterCutoffs, on: :create
```

Comment:
This validation only needs to happen on journal creation. It's validating that at the time of creation in relation to `Time.current`, it's within the right time window. If we were to do this validation on `update` (e.g. at journal reconciling), we would be outside the window, so the validation would fail.